style/calendar.less

```diff
@@ -0,0 +1,125 @@
+// Sample bootstrap theme for react-calendar
```

Comment:
actually, not without including some webpack dependencies or writing something for our gulp styles task to handle:

```less
@import '~bootstrap/less/bootstrap.less';
@import '~bootstrap/less/theme.less';
```

I will shelf this for now, and work on other aspects of the calendar first